### DISCUSSION TITLE
Considering opening a Slack channel for this if anyone is interested i

### DISCUSSION THREAD

**Discussion by gbowne1** [gbowne1/reactsocialnetwork#137] [Announcements]
I was considering opening a Slack channel for this project if anyone is interested in joining 

Let me know if you think this should be a thing.  

Thanks :)

@manuel12 @pawel975 @Dylan-Clarry

**Comment by pawel975** [comment]
I'm in!

**Reply by gbowne1** [reply]
Ok.  Just didn't know if it might be useful outside of GitHub for this project considering we don't have many devs contributing to it yet.  But time will tell. :)

**Reply by gbowne1** [reply]
https://join.slack.com/t/reactsocialmediadev/shared_invite/zt-1qxkurpk1-xP5wRx3oaCauIXBmt1ToVA

**Reply by gbowne1** [reply]
I do see there is some GitHub integrations since the last time I used Slack

**Comment by manuel12** [comment]
Great idea, this should speed up communication!

**Reply by gbowne1** [reply]
GitHub is fine but realtime communications should be improved with Slack, and Slack has integrations that will also be useful the mor the project grows.

**Reply by gbowne1** [reply]
Slack channel is inactive.
